charts/spire/charts/spire-server/README.md

```diff
@@ -97,14 +97,11 @@ A Helm chart to install the SPIRE server.
 | telemetry.prometheus.podMonitor.namespace | string | `""` | Override where to install the podMonitor, if not set will use the same namespace as the spire-server |
 | tolerations | list | `[]` |  |
 | topologySpreadConstraints | list | `[]` |  |
-| tornjak.config.backend.dataStore.driver | string | `"sqlite3"` |  |
-| tornjak.config.backend.dataStore.file | string | `"/run/spire/data/tornjak.sqlite3"` |  |
-| tornjak.config.frontend.apiServerURL | string | `"http://localhost:10000"` |  |
-| tornjak.enabled | bool | `false` |  |
-| tornjak.image.pullPolicy | string | `"IfNotPresent"` |  |
-| tornjak.image.registry | string | `"ghcr.io"` |  |
-| tornjak.image.repository | string | `"spiffe/tornjak"` |  |
-| tornjak.image.version | string | `"latest"` |  |
+| tornjak.config.backend.apiServerURL | string | `"http://localhost:10000/"` | URL of the Tornjak APIs (backend). Since Tornjak Frontend runs in the browser, this URL must be accessible from the machine running a browser. |
```

Comment:
Still don't think tornjak.config.backend.apiServerURL is needed in the spire-server chart since its for the frontend to find the backend? Can all references to it be removed?

Reply:
I agree with the removal of the `backend` prefix, but we still need the `apiServerURL` parameter, so users can go directly to Tornjak APIs. For example via _port forward_ or _ingress_ (soon to be added). Backend instance can be stood up without the Frontend,(e.g. to make remote calls to Tornjak APIs)

Reply:
Its still not an option that needs to be configured though? nothings using that value?